fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java

```diff
@@ -745,22 +748,22 @@ public PreAggStatus visit(Expression expr, CheckContext context) {
 
         @Override
         public PreAggStatus visitAggregateFunction(AggregateFunction aggregateFunction, CheckContext context) {
-            return checkAggFunc(aggregateFunction, AggregateType.NONE, context);
+            return checkAggFunc(aggregateFunction, AggregateType.NONE, context, false);
         }
 
         @Override
         public PreAggStatus visitMax(Max max, CheckContext context) {
-            return checkAggFunc(max, AggregateType.MAX, context);
+            return checkAggFunc(max, AggregateType.MAX, context, true);
```

Comment:
I don't know why min/max should be different from other aggregation methods. Can you explain it?

Reply:
Because max value or min value is not affected by the number of keys.
For example:
There are two rowsets. One includes key values: [1, 2]，the ohter one includes key values:[2, 3].
The merge key values should be [1, 2, 3].
The value of max(max([1,2]),  max([2, 3]) and the value of max([1, 2, 3]) are all 3.
min is the same as max.
But, the other funcs like sum are not.
The value of sum(sum([1,2]),  sum([2, 3]) is 8 and the right value of sum([1, 2, 3]) should be 6.